lib/player.js

```diff
@@ -45,12 +56,13 @@ var Player = {
 	},
 
 	// save player initials
-	update_initials: function(socket) {
+	update_initials: function(socket, initials) {
 		if (!socket || !socket.player_id || !socket.initials){
```

Comment:
if you're taking in the initials, no need to check them here, or to have them set on the socket in the first place.